src/Log/LogEntry.jsx

```diff
@@ -66,7 +67,7 @@ function hrefReplacer(str) {
                 href={href}
                 key={index}
                 tabIndex={index}
-                onClick={() => shell.openItem(href)}
+                onClick={() => openUrl(href)}
```

Comment:
I see this is inherited from old architecture. This will call `shell.openExternal()`, does that work properly for all platforms?

Reply:
We already use the same function in other places: Clicking on the logo or on the about pane. It better works on all platforms, otherwise it is broken in those places too. ;-) But I double checked on Windows.